src/Augmentation/AugmentationHelpers.jsx

```diff
@@ -1644,8 +1642,8 @@ function initAugmentations() {
              into your wrist. A small jack in the chip allows you to connect it to a computer 
              and upload the assets.<br /><br />
              This augmentation:<br />
-             Lets the player start with {Money(1e6)} after a reset.<br />
-             Lets the player start with the BruteSSH.exe program after a reset.</>,
+             "Provides {Money(1e6)} after a reset." +
```

Comment:
It's valid JS but it won't work as intended. The reason this description is JSX is because there's money involved and I made a patch where all the money in the game is color-coded in the same format.
<img width="544" alt="Screen Shot 2021-06-17 at 2 47 39 PM" src="https://user-images.githubusercontent.com/79246165/122455967-fc40d880-cf7a-11eb-9d21-05c2bcafcdfc.png">